### DISCUSSION TITLE
brew install fails; curl error

### DISCUSSION THREAD

**Discussion by xrival** [Homebrew/discussions#3981] [Everyday usage]
Hello,

I have just installed brew on a new machine (MacBook Air with M2). It cannot install anything. For instance, I get the error below:

bash-3.2$  brew install wget
==> Downloading https://ghcr.io/v2/homebrew/core/gettext/manifests/0.21.1
curl: (22) The requested URL returned error: 503                              
Warning: Problem : HTTP error. Will retry in 1 seconds. 3 retries left.
curl: (22) The requested URL returned error: 503                              
Warning: Problem : HTTP error. Will retry in 2 seconds. 2 retries left.
curl: (22) The requested URL returned error: 503                              
Warning: Problem : HTTP error. Will retry in 4 seconds. 1 retries left.
curl: (22) The requested URL returned error: 503                              

Error: wget: Failed to download resource "gettext_bottle_manifest"
Download failed: https://ghcr.io/v2/homebrew/core/gettext/manifests/0.21.1

I did some search, but could not resolve the issue.
I just tried on an iMac (intel) and got the same result with brew update, so I guess this is not an M2 specific issue.

Any idea would be welcome. Thanks!

**Comment by Bo98** [comment]
This is a GitHub outage. Follow https://www.githubstatus.com/incidents/ljwdk6qml8z8 for updates.

**Comment by chriselrod** [comment]
Here to say that I am facing the same problem.

**Comment by xrival** [comment]
Good to know it was just an outage. I was investigating issues related to the new OS and new machine.
It now works again for me. Many thanks for the replies!

**Comment by stephan0h** [comment]
Had a similar issue today (being a total newbie) and doing a ```export CURL_CA_BUNDLE=/etc/ssl/certs/ca-bundle.crt``` before brew install solved it for me ...
